db/models/aggregate_functions.py

```diff
@@ -215,12 +215,11 @@ def update_application_and_related_form(
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
     # updating project name
-    if form_name == "project-information":
+    if form_name == "project-information" or "gwybodaeth-am-y-prosiect":
```

Comment:
Since we're getting the form in any case could we just lookup the key and not check the `form_name` as the key should be unique?

Reply:
We do not get a key for the form name, we only get keys for the questions/ fields, so we have to validate the form name manually (in english and welsh)

Reply:
Can we add a TODO to extract this update title logic into a separate function?

Reply:
the project name update logic has now been moved to a separate function (seemed much easier just to do it here than create a separate BAU and change down the line as it was a quick copy paste pretty much 👍🏾)

Reply:
So the reasoning here is that the form_name comes in as metadata from the [formrunner](https://github.com/communitiesuk/funding-service-design-application-store/blob/main/api/routes/application/routes.py#:~:text=%22form_name%22%3A%20request_json%5B%22metadata%22%5D.get(%22form_name%22)%2C). We only want to search for the field if the form that contains that field is being updated.